packages/Operators/src/Search/DTK_ParallelSearch.cpp

```diff
@@ -250,27 +250,33 @@ void ParallelSearch::search(
 	// that were found on one process but missed on another.
 	std::sort( import_missed.begin(), import_missed.end() );
 	std::sort( import_found.begin(), import_found.end() );
-	Teuchos::Array<EntityId> false_positive_missed(
-	    import_missed.size() + import_found.size() );
+	Teuchos::Array<EntityId> false_positive_missed( import_missed.size() );
 	auto false_positive_end = 
 	    std::set_intersection( import_missed.begin(), import_missed.end(),
 				   import_found.begin(), import_found.end(),
 				   false_positive_missed.begin() );
 
+  // Create a unique list of missed entities
+	std::sort( import_missed.begin(), import_missed.end() );
+  auto import_missed_unique_end = std::unique(
+      import_missed.begin(), import_missed.end());
+  import_missed.resize(std::distance(import_missed.begin(), import_missed_unique_end));
+
 	// Create a list of missed entities without the false positives.
 	d_missed_range_entity_ids.resize( num_import_missed );
 	auto missed_range_end = std::set_difference( 
 	    import_missed.begin(), import_missed.end(),
 	    false_positive_missed.begin(), false_positive_end,
 	    d_missed_range_entity_ids.begin() );
-
-	// Create a unique list of missed entities without the false positives.
-	std::sort( d_missed_range_entity_ids.begin(), missed_range_end );
-	auto missed_range_unique_end = std::unique(
-	    d_missed_range_entity_ids.begin(), missed_range_end );
 	d_missed_range_entity_ids.resize(
 	    std::distance(d_missed_range_entity_ids.begin(),
-			  missed_range_unique_end) );
+			  missed_range_end) );
+
+#if HAVE_DTK_DBC
```

Comment:
What do you mean? Before I changed the order of ```unique``` and ```set_difference``` this would throw when using 6 processors. Now it doesn't throw anymore.